Source/Core/Core/IOS/ES/Formats.cpp

```diff
@@ -67,6 +67,12 @@ bool Content::IsOptional() const
   return (type & 0x4000) != 0;
 }
 
+bool operator==(const Content& lhs, const Content& rhs)
```

Comment:
This should probably have an `operator!=` like:

```cpp
bool operator!=(const Content& lhs, const Content& rhs)
{
  return !operator==(lhs, rhs);
}
```

to keep operations symmetrical.